k8s/crd-samples/deployment-cstor-pool.yaml

```diff
@@ -29,10 +29,14 @@ spec:
           mountPath: /dev     
         - name: tmp
           mountPath: /tmp
-          mountPropagation: Bidirectional
+#         mountPropagation: Bidirectional
+        lifecycle:
+          postStart:
+            exec:
+              command: ["/bin/sh", "-c", "sleep 15"]
 
       - name:    cstor-pool-mgmt
-        image: openebs/cstor-pool-mgmt:ci
+        image: gkganeshr/cstor-pool-mgmt:v1
```

Comment:
Are we sure we dont want to use openebs as the repo?

Reply:
Since I dont have credentials to push to openebs repo, I have mentioned this up,
```
Special notes for your reviewer: The image name here will be changed to openebs once the business logic pr is raised, reviewed and merged in openebs/maya.
```

Reply:
The same comments can be put in this yaml please!!!

Reply:
Since this is WIP and we have decided to move them to openebs/elves, I'll address the review comments in a separate PR. @kmova  @AmitKumarDas